### PR TITLE
Upgrade webpack-cli to 5.1.4

### PR DESCRIPTION
**Pull request by scherztc**
Fixes Dependabot's PR #207 

Since we moved off the webpacker gem there is still a need for the webpack-cli from 4.10.0 to 5.1.4.
To confirm that jsbundler is successful, we need to be able to run this node command:

node --trace-deprecation node_modules/webpack/bin/webpack.js --mode=none   

Use this command to test this PR in your local environment

### CODE REVIEW

**Comment by scherztc** [comment]
https://github.com/rails/jsbundling-rails/blob/main/docs/switch_from_webpacker.md

**Comment by scherztc** [comment]
https://webpack.js.org/migrate/5/

File: package.json
Comment (Janell-Huyck) [inline package.json:1]: [module-name] is intended as a placeholder for an actual module name that we want to be excluded from browser builds.  If we don't have a particular module-name to put in there, we should be able to remove that entire "browser" section.

**Review by Janell-Huyck — CHANGES_REQUESTED** [review]
I cannot deploy this branch.  I am getting the following errors:
...
      01 /opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/vendor/thor/lib/thor.rb:392:in `dis…
      01 /opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/cli.rb:34:in `dispatch'
      01 /opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/vendor/thor/lib/thor/base.rb:485:in…
      01 /opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/cli.rb:28:in `start'
      01 /opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/exe/bundle:45:in `block in <top (required)>'
      01 /opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/friendly_errors.rb:117:in `with_fri…
      01 /opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/exe/bundle:33:in `<top (required)>'
      01 /home/apache/.rbenv/versions/3.2.2/bin/bundle:25:in `load'
      01 /home/apache/.rbenv/versions/3.2.2/bin/bundle:25:in `<main>'
      01 Tasks: TOP => assets:precompile => javascript:build => javascript:install
      01 (See full trace by running task with --trace)
#<Thread:0x00000001067df5f8 /Users/huyckjl/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/sshkit-1.21.5/lib/sshkit/runners/parallel.rb:10 run> terminated with exception (report_on_exception is true):
/Users/huyckjl/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/sshkit-1.21.5/lib/sshkit/runners/parallel.rb:15:in `rescue in block (2 levels) in execute': Exception while executing as apache@libappstest.libraries.uc.edu: rake exit status: 1 (SSHKit::Runner::ExecuteError)
rake stdout: yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
rake stderr: error webpack-cli@5.1.4: The engine "node" is incompatible with this module. Expected version ">=14.15.0". Got "12.22.12"
error Found incompatible module.
rake aborted!
jsbundling-rails: Command install failed, ensure yarn is installed
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/jsbundling-rails-1.2.1/lib/tasks/jsbundling/build.rake:6:in `block (2 levels) in <main>'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/cli/exec.rb:58:in `load'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/cli/exec.rb:58:in `kernel_load'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/cli/exec.rb:23:in `run'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/cli.rb:492:in `exec'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundler/cli.rb:34:in `dispatch'
/opt/webapps/aaec/shared/vendor/bundle/ruby/3.2.0/gems/bundler-2.4.10/lib/bundle
...

**Comment by scherztc** [comment]
error webpack-cli@5.1.4: 
The engine "node" is incompatible with this module. Expected version ">=14.15.0". Got "12.22.12" error

**Pull request closed without merging**